go.mod

```diff
@@ -28,7 +30,7 @@ require (
 	google.golang.org/api v0.176.1
 	gopkg.in/yaml.v3 v3.0.1
 	k8s.io/api v0.29.3
-	k8s.io/apimachinery v0.29.3
+	k8s.io/apimachinery v0.30.0
 	k8s.io/client-go v0.29.3
```

Comment:
The title says "Bump k8s.io/client-go" but we're not bumping client-go?

Reply:
Good catch - fixed in 4b8f88adbb259b81b4d6c40b886053254b48dec2